Source/ACE/Entity/Player.cs

```diff
@@ -953,6 +953,18 @@ public void Logout(bool clientSessionTerminatedAbruptly = false)
             }
         }
 
+        public void SendToMarketPlace()
+        {
+            // TODO: link to Town Network marketplace portal destination in db, when db for that is finalized and implemented.
+            // private static readonly Position marketplaceDrop = new Position(23855548, 49.206f, -31.935f, 0.005f, 0f, 0f, -0.7071068f, 0.7071068f); // PCAP verified drop
+            // Session.Player.Teleport(marketplaceDrop);
+
+            // TODO: check current position of player against recorded position at time of command issuance, if they don't match, send the you've moved too far error (and thunk sound?).
+            // Enum_0498 of StatusMessageType1 ??
+            // Not sure what kind of tolerance there was with this check in respect to how far from initial start point one could move..
+            Session.Player.Teleport(new Position(23855548, 49.206f, -31.935f, 0.005f, 0f, 0f, -0.7071068f, 0.7071068f));

```

Comment:
This should still be a static position _somewhere_.  For lack of a better place, static values inside the Position class itself is a good place.  Also, you're already in the Player object so no need to do Session.Player.  Ideally, this line would be:
Teleport(Position.Marketplace);